sound/soc/sof/ipc3-topology.c

```diff
@@ -2473,11 +2473,6 @@ static int sof_ipc3_tear_down_all_pipelines(struct snd_sof_dev *sdev, bool verif
 	if (ret < 0)
```

Comment:
- typo in commit:   "pieplines"
- perhaps also "left over" -> "leftover"
- extra linefeed between fixes and signed-off lines